README.md

```diff
@@ -67,25 +67,32 @@ Add event listeners (init, disconnect, signIn):
 
 ```ts
 near.on("init", () => {
-   // your code
+  // your code
 });
 ```
 
 Interact with smart contract:
 
 ```ts
-near.getContract().callContract("getMessages", []).then(messages => {
-  console.log(messages);
-});
+near
+  .getContract()
+  .callContract("getMessages", [])
+  .then((messages) => {
+    console.log(messages);
+  });
 ```
 
 ## Example Integration
 
-A variation of the [guest-book](https://github.com/near-examples/guest-book/)  example project can be found in the `example` directory. You can use this to gain a concrete understanding of how to integrate this package into your own dApp.
+A variation of the [guest-book](https://github.com/near-examples/guest-book/) example project can be found in the `example` directory. You can use this to gain a concrete understanding of how to integrate this package into your own dApp.
 
+<<<<<<< HEAD
```

Comment:
It looks like some conflicts are still present here. As mentioned previously however, changes from `npm run lint:fix` should be done as part of another PR. The goal of this PR is just to enable ESLint without impacting the entire codebase